### PR TITLE
Clean-up build

### LINKED ISSUE #24: Set up github cache for builds

### PR DESCRIPTION
**Pull request by devpow112**
Sets explicit versions on a few actions used, clean-up some environment variable stuff and remove android emulators. Resolves #24.

### CODE REVIEW

**Comment by devpow112** [comment]
Cache is too big. Won't work. Will need to remove cache

File: Dockerfile [hunk 1]
```diff
@@ -62,13 +65,14 @@ RUN export DEBIAN_FRONTEND='noninteractive' && \
     sdkmanager \
       "platforms;android-22" \
       "platforms;android-${ANDROID_SDK_PLATFORM_VERSION}" \
```
Comment (devpow112) [inline Dockerfile:67]: I think once we go back to only installing 1 platform it may just fit in the cache

Comment (devpow112) [reply Dockerfile:54]: Cache works now. Managed to uninstall  android emulators which get installed by default.

File: Dockerfile [hunk 2]
```diff
@@ -58,17 +40,28 @@ RUN export DEBIAN_FRONTEND='noninteractive' && \
...
+    sdkmanager --update && \
     sdkmanager \
-      "platforms;android-22" \
+      'platforms;android-22' \
       "platforms;android-${ANDROID_SDK_PLATFORM_VERSION}" \
-      "build-tools;${ANDROID_SDK_BUILD_TOOLS_VERSION}" \ 
+      "build-tools;${ANDROID_SDK_BUILD_TOOLS_VERSION}" \
       platform-tools \
       ndk-bundle && \
+    sdkmanager --uninstall emulator && \
```
Comment (devpow112) [inline Dockerfile:58]: This saves us around 700 MB in size. Since the emulators aren't used they can be removed.

Comment (AWoloszyn) [reply Dockerfile:58]: neat! I wish we should just NOT install them in the first place, but oh well.

**Review by AWoloszyn — APPROVED** [review]

**Pull request merged**